src/providers/mod.rs

```diff
@@ -156,6 +175,14 @@ fn write_ssh_keys(user: User, ssh_keys: Vec<PublicKey>) -> Result<()> {
         .chain_err(|| format!("failed to remove file {:?}", file_path.display()))?;
     }
 
+    let name = user.name().to_string_lossy();
+    let path = file_path.to_string_lossy();
+    let message = format!("wrote ssh authorized keys file for user:{}", name.as_ref());
+    let sent = write_structured_journald_entry(logging::Priority::Info, &message, &name, &path);
+    if let Err(e) = sent {
+        warn!("failed to write structured journald entry: {}", e);
+    }
```

Comment:
We already warn in `write_structured_journald_entry`, right? So we could just call it and use `?` here instead. (I guess we could change that function to be infallible now, though meh... nice to just leave it like that too in case we do add errors in the future.)

Reply:
Let's make it infallible, we don't plan to fail the pubkeys flow because of journald.